File: test/fixtures/output/elixir/httpoison/multipart-data.ex
HTTPoison.request(
  :post,
  "http://mockbin.com/har",
  {:multipart, [{:file, "hello.txt", {"form-data", [{"name", "foo"}, {"filename", "hello.txt"}]}, [{"content-type", "multipart/form-data"}]}]},
  [{"content-type", "multipart/form-data"}]
)

File: src/targets/node/axios/fixtures/multipart-data.js
const axios = require('axios').default;

const options = {
  method: 'POST',
  url: 'http://mockbin.com/har',
  headers: {'content-type': 'multipart/form-data; boundary=---011000010111000001101001'},
  data: '-----011000010111000001101001\r\nContent-Disposition: form-data; name="foo"; filename="hello.txt"\r\nContent-Type: text/plain\r\n\r\nHello World\r\n-----011000010111000001101001\r\nContent-Disposition: form-data; name="bar"\r\n\r\nBonjour le monde\r\n-----011000010111000001101001--\r\n'
};

try {
  const { data } = await axios.request(options);
  console.log(data);
} catch (error) {
  console.error(error);
}